storage/plugins/book-club/src/PollController.php

```
@@ -0,0 +1,1170 @@
+        $optionIds = $body['options'] ?? [];
+        if (!is_array($optionIds)) {
+            $optionIds = [];
+        }
+        $optionIds = array_values(array_unique(array_map('intval', $optionIds)));
+        if (count($optionIds) < 2) {
+            $this->flash('error', __('Seleziona almeno due proposte da mettere in votazione.'));
+            return $this->redirect($response, '/book-club/' . $slug);
+        }
+
+        $states = $this->repo->workflowStates($club);
+        $votingState = Repo::votingStateKey($states);
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'App\Plugins\BookClub\Repo' in method 'create'. (undefined)

(StaticAccess)